client/gui_rpc_server_ops.cpp

```diff
@@ -775,7 +775,8 @@ void handle_get_project_config_poll(GUI_RPC_CONN& grc) {
         );
     } else {
         const char *p = grc.get_project_config_op.reply.c_str();
-        const char *q = strstr(p, "<project_config");
+        const char *q = strstr(p, "<project_config>");
```

Comment:
This will remove attribute support from this tag. If, for whatever reason, we decide to put an attribute to that tag it will not be recognized with this change.